.gitignore

```diff
@@ -36,3 +36,6 @@ TAGS
 # Visual Studio Code project files
 /.vscode/
 /.ccls-cache/
+
+# Qt Creator project files
+CMakeLists.txt.user
```

Comment:
Should only appear at the root level, right? So please add `/` in the front.

Reply:
Done.